client/example/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "urql-client-example",
```

Comment:
I'd rather put this example in the main examples folder. 😅just to have them all in one place. It may also be better to generate this client folder only on demand so it doesn't confuse anyone

Reply:
How would you generate that on demand?

Reply:
A prepublish script that creates the JSON and the folder which can probably just point to the dist folder (../dist/x.js)